ros2_utils.bash

```diff
@@ -1,5 +1,23 @@
 #!/usr/bin/env bash
 
+# ROS 2 run
+
+function rrun {
```

Comment:
please rename to `rnrun` and put in the `Nodes` section

Reply:
I actually didn't know that `ros2 run` existed, I thought only `ros2 node run`. But they seem to me like they do the same thing so what do you think about using `ros2 node run` since we already have a section for it?

Reply:
I have really no clue how I got that confused, you're right `ros2 node run` does not exist -> let's go with your proposal of for `rrun`

Reply:
Thank you for your consideration.
`rnrun` has been renamed into `rrun`.
I also merged `main` to make sure there were no conflicts.